apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/derived/DerivedTableInterface.java

```diff
@@ -24,4 +30,17 @@
 
     boolean delete(T t);
 
+    default DerivedTableData<T> getAllByDbId(MinMaxDbId minMaxDbId, int limit) throws SQLException {
+        throw new UnsupportedOperationException("GetAll is not supported");
```

Comment:
These default methods were implemented in DerivedDbTable class, please remove default implementations

Reply:
Removed completely